packages/core/content-manager/admin/src/layout.tsx

```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 
 import { Page, useGuidedTour, Layouts } from '@strapi/admin/strapi-admin';
 import { useIntl } from 'react-intl';
-import { Navigate, Route, Routes, useLocation, useMatch } from 'react-router-dom';
+import { Navigate, Outlet, Route, Routes, useLocation, useMatch } from 'react-router-dom';
```

Comment:
```suggestion
import { Navigate, Outlet, useLocation, useMatch } from 'react-router-dom';
```
You can also remove `routes` from the line down below too